src/app/styl/views/filter_bar.styl

```diff
@@ -24,16 +24,18 @@
         font-size: 1.4em
         text-transform: uppercase
         border-bottom: 3px solid transparent
+        opacity: 0.4
 
         &:hover
-            color: $FilterBarTextHover
+            opacity: 0.9
```

Comment:
Ok oh, I see. But then you're forcing a set opacity, instead of leaving a theming choice